htdocs/fourn/class/fournisseur.facture-rec.class.php

```diff
@@ -852,8 +854,8 @@ public function fetch_lines()
 				$line->total_localtax2          = $objp->total_localtax2;
 				$line->total_ttc                = $objp->total_ttc;
 				$line->product_type             = $objp->product_type;
-				$line->date_start               = $this->db->jdate($objp->date_start);
-				$line->date_end                 = $this->db->jdate($objp->date_end);
+				$line->date_start               = $objp->date_start; // integer (not a date))
```

Comment:
$objp->date_start is a string from the sql answer.
We must use $this->db->jdate() to transformthe text date 'YYYYMM-DD-HH...' into a timestamp.

Reply:
@eldy this is not a date but a yes/no

<img width="1564" height="320" alt="Capture d’écran 2025-11-28 à 16 39 13" src="https://github.com/user-attachments/assets/2b9d4871-e081-4d56-93c0-bcbc17f3f7a0" />

Reply:
@eldy These fields will need to be renamed as in the customer invoice templates.

```
date_start => date_start_fill
date_end => date_end_fill
```

Reply:
@eldy permet moi d'avoir raison parfois ! ;-)
après 20 ans je fais des tests dans tous ls sens ! 😄